README.md

```diff
@@ -0,0 +1,44 @@
+
+A simple sample of microservice on spring-boot.
+
+## Description
+
+User service provides endpoints to receive **data related to users**.<br/>
+
+It is connected with another microservice via **service registry pattern**. The Service Registry is a database for microservice instances. 
+When the client service need to access internal services, then it will query from Service Registry and access them.
+Here implementation is done via Netflix Eureka.
+
+<p align="center">
+  <img src="http://1.bp.blogspot.com/-dXDcC3rnVeU/V7J79R5XxII/AAAAAAAADOQ/K53Ebgvj0uweGLXDeVKanQa3uFxxxRCvwCK4B/s1600/img.png" width="420" height="270">
+</p>
+
+**Circuite breaker pattern** is also implemented there. The basic idea behind the circuit breaker is very simple. 
```

Comment:
@eyakauleva  no need to explain the pattern. instead explain why you need it

Reply:
fixed